build/Microsoft.DotNet.Cli.Prepare.targets

```diff
@@ -1,5 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="Microsoft.DotNet.Cli.Monikers.props" />
```

Comment:
It's kind of weird for Prepare to import this props file, but nothing Prepare actually uses it.

I think instead we should either:

1. Import it where it is needed (i.e. 'package')
2. Import it from the root build.proj.